src/views/Authentications.js

```diff
@@ -101,10 +106,16 @@ export class Authentications extends React.PureComponent {
     showAuthWindow = (id) => () => {
         getAuthEditUrl(id)
             .then(({body}) => {
-                openAuthWindow(body.data.popupUrl);
+                openAuthWindow(`${body.data.popupUrl}&${this.state.params}`);
```

Comment:
I guess `body.data.popupurl` already has some query parameters, right?

Reply:
yes, it has always `?code=...`